qcodes/utils/types.py

```diff
@@ -8,22 +8,22 @@
 
 numpy_concrete_ints = (np.int8, np.int16, np.int32, np.int64,
                        np.uint8, np.uint16, np.uint32, np.uint64)
-numpy_non_concrete_ints_instantiable = (np.int, np.int_, np.uint)
+numpy_non_concrete_ints_instantiable = (np.int_, np.uint)
 numpy_non_concrete_ints = \
     numpy_non_concrete_ints_instantiable + (np.integer,)
 
 numpy_concrete_floats = (np.float16, np.float32, np.float64)
```

Comment:
add `np.double` which is normally (always?) an alias for `np.float64`   

Also consider adding `np.longdouble` which is a `80` or `128` bit float